exporters/tests/test_failure_exporter.py

```diff
@@ -126,6 +108,7 @@ def mock_jira_connect(self):
         projects=None,
     )
     collector.search_issues()
+    # TODO test?
```

Comment:
This test is testing something? There is  o assert

Reply:
Got it. A suggestion would be to check if nothing was raised, I believe we can do it with something like this https://github.com/mateusoliveira43/cly/blob/main/tests/unit/cly/test_config.py#L36